tests/test_bad_fonts_nonrecursive.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
I don’t like the duplication of the scripts, but this can be handled differently later.